README.md

```diff
@@ -261,82 +271,56 @@ Renders the widget to the DOM, and passes control back to your app through succe
 
 - `options`
   - `el` - CSS selector which identifies the container element that the widget attaches to.
-- `success` *(optional)* - Function that is called when the user has completed an authentication flow. If an [OpenID Connect redirect flow](#openid-connect) is used, this function can be omitted.
+- `success` *(optional)* - Function that is called when the user has completed an authentication flow. If an [OpenID Connect redirect flow](#openid-connect) is used, this function can be omitted. See below for more details.
 - `error` *(optional)* - Function that is called when the widget has been initialized with invalid config options, or has entered a state it cannot recover from. If omitted, a default function is used to output errors to the console.
 
+#### success
+
+Function that handles non-error events, including the completion of a successful authentication flow by a user. Also handles other flows such as reset password and self-registration flows.
+
+_OIDC vs Non-OIDC Configurations:_
+
+New integrations are recommended to use OIDC as it uses a more lightweight REST-based protocol and has more widespread usage. Less common is the need for non-OIDC integrations in self-hosted applications, however one example of this usage is when using the Okta-hosted sign-in widget to redirect a user to the Okta Dashboard after authentication.
+
+**`res` parameter:**
```

Comment:
The style for `h4` (`####`) and bold looks very close and may confuse people `res` is at same layer as `success`.
Would be more proper to change to
`/res parameter/` 
the goal is to make more clear that we're talking about the parameter of `success` callback.